x-pack/plugins/apm/server/routes/services/get_services_detailed_statistics/get_service_aggregated_transaction_detailed_statistics.ts

```diff
@@ -186,7 +175,7 @@ export async function getServiceAggregatedTransactionDetailedStats({
             y: calculateThroughputWithRange({
               start,
               end,
-              value: dateBucket.total_doc.value,
+              value: dateBucket.doc_count,
```

Comment:
you can also remove the `total_doc` agg no?

Reply:
Thanks! I missed it, will remove it